criu/netfilter.c

```diff
@@ -299,7 +299,25 @@ int nftables_lock_connection(struct inet_sk_desc *sk)
 
 int nftables_get_table(char *table, int n)
 {
-	if (snprintf(table, n, "inet CRIU-%d", root_item->pid->real) < 0) {
+	int ret;
+
+	switch(dump_criu_run_id[0]) {
+	case 0:
+		/* This is not a restore.*/
+		ret = snprintf(table, n, "inet CRIU-%s", criu_run_id);
```

Comment:
What does `"This is not a restore"` mean here?

Reply:
If doing a dump the name of the table is based on the current criu_run_id and not on the one from the previous dump. It means, that this is not a restore but a dump.